pkg/agent/route/route_linux.go

```diff
@@ -459,7 +459,7 @@ func (c *Client) syncIPTables() error {
 			antreaPodIP6Set,
 			localAntreaFlexibleIPAMPodIP6Set,
 			antreaNodePortIP6Set,
-			config.VirtualNodePortDNATIPv4,
```

Comment:
wonder why ipv6 and dual-stack test can pass without the basic route

Reply:
Only for proxyAll.